core/src/main/java/bio/terra/pearl/core/service/workflow/RegistrationService.java

```diff
@@ -39,13 +45,19 @@ public class RegistrationService {
     private EventService eventService;
     private ObjectMapper objectMapper;
 
+    private static final String CHARACTERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+    private static final String GOVERNED_USER_INDICATOR = "-proxied-";
```

Comment:
shortening for easier readability in admin tool
```suggestion
    private static final String GOVERNED_USERNAME_INDICATOR = "-prox-";
```